### ISSUE TITLE
issue with uploading an image

### ISSUE THREAD

**Issue by sj134629** [Montana-Media-Arts/120_CreativeCoding#223]
I am having a hard time with up an image i went through the instructions and made sure that the code is right but it is still not working. It keeps saying that its loading.
![Sketch](https://user-images.githubusercontent.com/46580649/56833770-a5bd3100-682c-11e9-977d-be087a077ec8.png)

**Comment by ECPina** [comment]
Yeah, I had the same problem with audio and images. I'm still digging through the web to find a solution.

**Comment by tjschweitzer** [comment]
@sj134629 if the file doesn't exist (misspelled, wrong file type or just like need capitals) it will never finish the preload function.  Hope this helps.

**Comment by sj134629** [comment]
![Sketch7](https://user-images.githubusercontent.com/46580649/56867128-01a7c700-699f-11e9-812f-f0dea721dbbe.png)
this is what I have. I added the ./ to it an it did the same thing.

**Comment by sj134629** [comment]
@ECPina it works in Firefox. Not in chrome.

**Comment by ECPina** [comment]
@sj134629 Yeah, I figured it out after some tinkering and reading up on google’s policies for developers.

**Comment by mkennedy87** [comment]
I had this problem too on chrome but it worked as soon as I remembered to run the local server, haven't had any problems since
